src/output.jl

```diff
@@ -172,41 +172,32 @@ end
 """Checks output folders to determine a 4-digit run id number."""
 function get_run_id_path(S::ModelSetup)
 
-    @unpack output,outpath,get_id_mode = S.parameters
+    @unpack output,outpath,get_id_mode,path = S.parameters
 
     if output
-        runlist = filter(x->startswith(x,"run"),readdir(outpath))
-        existing_runs = [parse(Int,id[4:end]) for id in runlist]
+        
+        pattern = r"run_\d\d\d\d"               # run_???? in regex
+        runlist = filter(x->startswith(x,pattern),readdir(path))
```

Comment:
Why is this not `outpath` anymore? If you rename this to `path` here then this refers to what was used to be the `initpath` in `default_parameters.jl`?

Reply:
This at least wasn't me renaming something. ShallowWaters.jl had a single function `get_run_id_path` that returned (if I understood correctly) the run_id and the run_path. In SpeedyWeather.jl this was accomplished in two functions`get_run_id` and `create_output_folder` (technically three if you count `run_id_to_string`, which also happens in the single function in ShallowWaters). All I really did was combine the functions from SpeedyWeather into the single function in ShallowWaters by cutting and pasting, so I didn't really think too hard on what the variable names were

Reply:
See below comments, I now understand what you're referring to